strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/providers/io/RepositoryPathTest.java

```diff
@@ -12,28 +15,45 @@
 import org.hamcrest.CoreMatchers;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestExecutionListeners;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 /**
  * @author Przemyslaw Fusik
  * @author Pablo Tirado
  */
+@SpringBootTest
+@ActiveProfiles(profiles = "test")
+@ContextConfiguration(classes = StorageApiTestConfig.class)
+@TestExecutionListeners(listeners = { CacheManagerTestExecutionListener.class },
+                        mergeMode = TestExecutionListeners.MergeMode.MERGE_WITH_DEFAULTS)
 public class RepositoryPathTest
 {
 
-    private static final Path REPOSITORY_BASEDIR = Paths.get("target/strongbox-vault/storages/storage0/releases").toAbsolutePath();
+    private static final String STORAGE_ID = "storage0";
+
+    private static final String REPOSITORY_ID = "releases";
+
+    private Path repositoryBasePath;
 
     private RepositoryDto repository;
 
     private LayoutFileSystem repositoryFileSystem;
 
+    @Inject
```

Comment:
it's seems not really good to setup the Spring Application context only to have `ProperotesBooter` instance here. This test just need a sandbox folder under `target` folder, we can try using [this](https://stackoverflow.com/a/28674230/6825250) solution based on classpath. This way there will be no need for `PropertiesBooter` here.